test/specs/components/pagination/paging_collection_spec.js

```diff
@@ -46,12 +46,22 @@ define(['jquery',
             };
 
             beforeEach(function () {
-                collection = new PagingCollection([], {state: {perPage: 10}});
+                collection = new PagingCollection([], {state: {pageSize: 10}});
                 collection.url = '/test';
                 server.isZeroIndexed = false;
                 server.count = 43;
             });
 
+            it('correctly merges state and queryParams in the extend statement', function () {
+                var MyPagingCollection = PagingCollection.extend({
+                    state: {pageSize: 25},
+                    queryParams: {pageSize: 'per_page'}
+                });
+                var newCollection = new MyPagingCollection([]);
```

Comment:
Nitpick: there should only be 1 ```var``` per function, using commas to define multiple variables.